examples/tutorial/tutorial_notebooks/cluster/exercise_5.ipynb

```diff
@@ -13,55 +13,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Exercise 5: Executing on a cluster environment\n",
+    "# Exercise 5: Setting up cluster environment\n",
```

Comment:
If we deleted exercise 4, this should be renamed to exercise 4 (and all subsequent exercises should be modified).

Reply:
We added exercise 4 in this commit, so I've bumped all the notebooks number from 5 and onwards